### PR TITLE
Added an API doc page for pre-built workflow

### PR DESCRIPTION
**Pull request by y27choi**
### Linked issue(s):
Fixes KOL-2711

### What change does this PR introduce and why?
Added a doc for pre-built 2D OD APIs
Borrowed some code/comments from [this PR](https://github.com/kolenaIO/kolena/pull/82)

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

**Review by ly-kolena — APPROVED** [review]

File: kolena/_experimental/object_detection/workflow.py [hunk 1]
```diff
@@ -51,10 +65,16 @@ def __post_init__(self):
...
+
     bboxes: List[ScoredLabeledBoundingBox]
+    """
+    The inference [`ScoredLabeledBoundingBox`][kolena.workflow.annotation.ScoredLabeledBoundingBox]es associated with
+    an image.
+    """
 
 
-_workflow, TestCase, TestSuite, Model = define_workflow(
+_workflow, _TestCase, _TestSuite, _Model = define_workflow(
```
Comment (ly-kolena) [inline kolena/_experimental/object_detection/workflow.py]: Looks like this is causing import issues.

File: docs/overrides/.icons/kolena/drive-time-20.svg
Comment (gordonhart) [inline docs/overrides/.icons/kolena/drive-time-20.svg:1]: Are these used by the new docs page?

**Review by gordonhart — APPROVED** [review]
Looks great! I'll follow up with some minor reorgs/polish before today's release.

File: kolena/_experimental/object_detection/workflow.py (hunk 1 above)
Comment (y27choi) [reply kolena/_experimental/object_detection/workflow.py]: I think you are right. Thanks for spotting this.

**Pull request merged**